variables.tf

```diff
@@ -112,7 +96,13 @@ variable "managed_identities" {
     user_assigned_resource_ids = optional(set(string), [])
   })
   default     = {}
-  description = "Managed identities to be created for the resource."
+  description = <<DESCRIPTION
+  Controls the Managed Identity configuration on this resource. The following properties can be specified:
+  
+  - `system_assigned` - (Optional) Specifies if the System Assigned Managed Identity should be enabled.
```

Comment:
Why we need the `system_assigned` boolean at line 95 ? 
Why this is a object instead of a list ?

Reply:
the avm enforced this for using managed identities i initially removed this but it looks like they have a check that enforces this structure when using managed identities, I have removed the ability to set system assigned though in the local.tf file

Reply:
this might be good feedback to provide avm for instances where this is enforced

Reply:
https://github.com/Azure/terraform-azurerm-avm-ptn-aks-production/pull/40/files#r1553715163